### ISSUE TITLE
Add replace filter

### ISSUE THREAD

**Issue by hatifnatt** [HeavyHorst/remco#37]
There is no `replace` filter in Django template and it actually can be very useful for some basic string manipulation. Due inability to pass multiple variables to Django filer (as I can see from my limited experience) it's tricky to implement flexible port for JavaScript `str.replace()` function.

confd implement `replace` function by simply aliasing Go's `strings.Replace` function.

My not really good example with a basic filter which only replace one character.
JS filer
```js
var pattern = Param.slice(0,1);
var replacement = Param.slice(1,2);
In.replace(pattern, replacement);
```
Sample data and usage
```
v = user@domain.tld
v|replace_char:'@/'|base:'-1'

result: domain.tld
```

**Comment by HeavyHorst** [comment]
There is a replace function that is just aliasing strings.Replace like confd does: https://github.com/HeavyHorst/remco/blob/master/pkg/template/template_funcs.go

https://heavyhorst.github.io/remco/template/template-functions/

**Comment by hatifnatt** [comment]
Ahh, my bad! Missed it somehow, I think because I looked for it in [filters sections of doc](https://heavyhorst.github.io/remco/template/template-filters/) and in [filter sources](https://github.com/HeavyHorst/remco/blob/master/pkg/template/template_filters.go). Thank you for pointing in right direction!

I fixed my template, one custom JS filter less. I noticed that for some reason confd don't use comma as function arguments delimiter. I do not mean that remco should act in the same way, just observation.
